docker-compose/sep24/docker-compose.yaml

```diff
@@ -21,7 +21,7 @@ services:
       - events.publisher.kafka.bootstrap_server=kafka:29092
       # data
       - data.type=postgres
-      - data.server=db:5432
+      - data.server=db:5440
```

Comment:
Why do we use 5440 instead of 5432?

Reply:
Some services were looking for the DB in port 5440, and if we define port 5432 on docker-compose.yaml the services will fail to connect to the DB

Reply:
Updated to use 5432